CMakeLists.txt

```diff
@@ -533,28 +503,23 @@ foreach(header ${HEADERS_TO_CHECK})
     string(TOUPPER ${header} tmp)
 
     string(REGEX REPLACE [./] "_" upper_header ${tmp})
-    check_include_file(${header} HAVE_${upper_header})
+    set(header_flag "HAVE_${upper_header}")
+    check_include_file(${header} ${header_flag})
+    if (${header_flag})
+        add_compile_definitions(${header_flag})
+    endif()
 endforeach()
 
 if (NOT HAVE_INTTYPES_H)
     message(FATAL_ERROR "Csound requires inttypes.h")
 endif()
 
-if (HAVE_EXECINFO_H)
-    add_definitions("-DHAVE_EXECINFO")
-endif()
-
-check_deps(USE_LRINT HAVE_LRINT)
-if(USE_LRINT)
-    add_definitions("-DUSE_LRINT")
-endif()
```

Comment:
Are these moved somewhere else or just deleted?

Reply:
Did these in batch with `FUNCTIONS_TO_CHECK`. But will probably have to do them individually if we want to specify which target to add them too.

Reply:
These should be added to libcsound target

Reply:
Moved to #1822